src/java.base/windows/native/libnet/TwoStacksPlainDatagramSocketImpl.c

```diff
@@ -22,7 +22,12 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-#include <malloc.h>
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+ * ===========================================================================
+ */
+ #include <malloc.h>
```

Comment:
Please don't change this line (no leading space).

Reply:
Removed the space.